src/pages/about.js

```diff
@@ -52,23 +52,54 @@ module.exports = React.createClass({
       aboutCopy: aboutCopy
     });
   },
+  renderTextAboutPage: function() {
+    var aboutCopy = this.state.aboutCopy;
+
+    return (
+      <div className="container additional-page">
+        <img className="heart-image icon-baseline" height="100" width="107" src="/assets/images/heart.ce7d2d59c757e1598e244e546426577c.svg"/>
+        <img className="heart-image icon-variant" height="100" width="107" src="/assets/images/pixel-heart.svg"/>
+        <div>
+          <img className="mozilla-watermark" src="/assets/images/mozilla.1068965acefde994a71c187d253aca2b.svg"/>
+          {aboutCopy}
+        </div>
+      </div>
+    );
+  },
+  renderVideoAboutPage: function() {
+    return (
+      <div className="container additional-page">
+        <div className="content-wrapper">
+          <img className="mozilla-watermark" alt="Mozilla Logo" src="/assets/images/mozilla.1068965acefde994a71c187d253aca2b.svg"/>
+        </div>
+        <video controls poster="/assets/images/donate-video-poster.jpg">
+          <source src="https://assets.mofoprod.net/fundraising/2017/fundraising-video-single-caption-480p.webm" type="video/webm" />
+          <source src="https://assets.mofoprod.net/fundraising/2017/fundraising-video-single-caption-480p.mp4" type="video/mp4" />
+          Your browser does not support the video tag.
+        </video>
+        <div className="content-wrapper">
+          <p>We are proudly non-profit, non-corporate and non-compromised. Thousands of people like you help us stand up for an open Web for all. We rely on donations to carry out our mission to keep the Web open and free. Will you give today?</p>
```

Comment:
we should probably pull this string using the en-US `additional_info` message key ([here](https://github.com/mozilla/donate.mozilla.org/blob/master/locales/en-US/messages.properties#L62)), using the same kind of code we're currenlty using here: https://github.com/mozilla/donate.mozilla.org/blob/0515fa5cf8f75ef37bdd3d563e40c33fac9fb239/src/pages/about.js#L16

Reply:
Yes, let’s localize this one — we will run the experiment in German too once the video is ready